api/database/factories/PoolFactory.php

```diff
@@ -33,17 +30,4 @@ public function definition()
             'user_id' => User::factory(),
         ];
     }
-
-    public function configure()
-    {
-        return $this->afterCreating(function (Pool $pool) {
-            $assets = CmoAsset::inRandomOrder()->limit(4)->get();
-            $classifications = Classification::inRandomOrder()->limit(3)->get();
-            $requirements = OperationalRequirement::inRandomOrder()->limit(2)->get();
-            $pool->essentialCriteria()->saveMany($assets->slice(0,2));
-            $pool->assetCriteria()->saveMany($assets->slice(2,2));
-            $pool->classifications()->saveMany($classifications);
-            $pool->operationalRequirements()->saveMany($requirements);
-        });
```

Comment:
Why take this out of PoolFactory? That makes using a Pool more difficult in phpunit tests, for example.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/1376#pullrequestreview-834242890

I'm not sure why it wasn't working. Want to hop on a call and check it out?